android/src/main/java/com/cloudwebrtc/webrtc/PeerConnectionObserver.java

```diff
@@ -465,6 +457,10 @@ public void onAddTrack(RtpReceiver receiver, MediaStream[] mediaStreams) {
           trackInfo.putBoolean("remote", true);
           params.putMap("track", trackInfo.toMap());
           sendEvent(params);
+
+          if ("audio".equals(track.kind())) {
```

Comment:
I have a question, when execute getUserMedia, a local audioTrack is created, should we also call `AudioSwitchManager.instance.start();`? In my memory, `onAddTrack/onTrack` should be triggered after the remote stream arrives.

Reply:
My first thought was, that it's used only for output control, missed that, will fix